src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,21 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int PERIOD_START_INDEX = 0;
+    private static final int PERIOD_END_INDEX = 1;
+    private static final int MIN_AGE = 35;
+    private static final String VALID_NATIONALITY = "Ukrainian";
+    private static final int VALID_PERIOD_IN_UKR = 10;
+
+    @Override
+    public boolean test(Candidate candidate) {
+        int periodInUkr = Integer.parseInt(candidate.getPeriodsInUkr().split("-")[PERIOD_END_INDEX])
+                - Integer.parseInt(candidate.getPeriodsInUkr().split("-")[PERIOD_START_INDEX]);
```

Comment:
```suggestion
        String[] periods = candidate.getPeriodsInUkr().split("-");
        int periodInUkr = Integer.parseInt(periods[PERIOD_END_INDEX])
                - Integer.parseInt(periods[PERIOD_START_INDEX]);
```
you are spliting 2 times